buildSrc/src/main/kotlin/org/zaproxy/zap/jflex.gradle.kts

```diff
@@ -6,7 +6,7 @@ import org.zaproxy.zap.tasks.JFlexToRstaTokenMaker
 val jflexClasspath by configurations.creating
 
 dependencies {
-    jflexClasspath("de.jflex:jflex:1.8.2")
+    jflexClasspath("de.jflex:jflex:1.9.1")
```

Comment:
Did you try regenerate the code too?

Reply:
Nope, happy to, what's the process for that?

Reply:
Updated file attached

Reply:
Now with spotless

Reply:
Needs further review (we should be doing those changes through `JFlexToRstaTokenMaker`, It might also be possible that some changes are no longer needed).

Reply:
Shall I skip the jflex bit for this PR?

Reply:
It's fine to update it anyway, if the tests pass the code is good to go.